README.md

```diff
@@ -82,6 +83,17 @@ docker run -p 8000:8000 wagtailcms
 
```

Comment:
README.md should explain how to install and run postgres locally, we don't want to be using the development environment postgres db.

Reply:
I believe the following needs to be updated on the README.md: